wls-basisdaten-service/src/test/java/de/muenchen/oss/wahllokalsystem/basisdatenservice/rest/wahltage/WahltageControllerIntegrationTest.java

```diff
@@ -0,0 +1,218 @@
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.cloud.contract.wiremock.AutoConfigureWireMock;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.transaction.annotation.Transactional;
+
+@SpringBootTest(classes = MicroServiceApplication.class, webEnvironment = SpringBootTest.WebEnvironment.MOCK)
+@AutoConfigureMockMvc
+@AutoConfigureWireMock
+@ActiveProfiles(profiles = { SPRING_TEST_PROFILE, SPRING_NO_SECURITY_PROFILE })
+public class WahltageControllerIntegrationTest {
+
+    @Value("${service.info.oid}")
+    String serviceID;
+
+    @Autowired
+    MockMvc api;
+
+    @Autowired
+    ObjectMapper objectMapper;
+
+    @Autowired
+    WahltageDTOMapper dtoMapper;
+
+    @Autowired
+    WahltagModelMapper modelMapper;
+
+    @Autowired
+    WahltageClientMapper wahltageClientMapper;
+
+    @Autowired
+    WahltagRepository wahltagRepository;
+
+    @Autowired
+    WahlvorschlagRepository wahlvorschlagRepository;
+
+    @Autowired
+    KandidatRepository kandidatRepository;
+
+    @AfterEach
+    void tearDown() {
+        SecurityUtils.runWith(Authorities.ALL_AUTHORITIES_DELETE_WAHLTAGE);
+        wahltagRepository.deleteAll();
+    }
+
+    @BeforeEach
+    void setup() {
+        WireMock.resetAllRequests();
+    }
+
+    @Nested
+    class GetWahltage {
+
+        @Test
+        void loadedFromExternal() throws Exception {
+
+            String requestDate = LocalDate.now().minusMonths(3).toString();
+
+            val eaiWahltage = createClientWahltageDTO(false);
+            WireMock.stubFor(WireMock.get("/wahldaten/wahltage?includingSince=" + requestDate)
+                    .willReturn(WireMock.aResponse().withHeader("Content-Type", "application/json").withStatus(HttpStatus.OK.value())
+                            .withBody(objectMapper.writeValueAsBytes(eaiWahltage))));
+
+            val request = MockMvcRequestBuilders.get("/businessActions/wahltage");
+
+            val responseFromController = api.perform(request).andExpect(status().isOk()).andReturn();
+            val responseBodyAsDTO = objectMapper.readValue(responseFromController.getResponse().getContentAsString(),
+                    de.muenchen.oss.wahllokalsystem.basisdatenservice.rest.wahltage.WahltagDTO[].class);
+
+            val expectedResponseBody = dtoMapper
+                    .fromListOfWahltagModelToListOfWahltagDTO(wahltageClientMapper.fromRemoteClientWahltageDTOtoListOfWahltagModel(eaiWahltage));
+
+            Assertions.assertThat(responseBodyAsDTO).containsExactlyInAnyOrderElementsOf(expectedResponseBody);
+        }
+
+        @Test
+        @Transactional
+        void externalDataIsPersisted() throws Exception {
+
+            String requestDate = LocalDate.now().minusMonths(3).toString();
```

Comment:
- Leerzeile entfernt
- Umstellung auf `final`

```suggestion
        void externalDataIsPersisted() throws Exception {
            val requestDate = LocalDate.now().minusMonths(3).toString();
```

Reply:
resolved 42209710dc768d8715816e674080af90d2d7f675

Reply:
Leerzeile ist noch da

Reply:
3d6b4734150e3fae3cc8d9d5ccce4a2e582218c0